webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src/index.js',
```

Comment:
You could use context property. It would allow you not to write the whole path relativly to the root folder. https://webpack.js.org/configuration/entry-context/#context. For example we define: context: path.resolve(__dirname, '/src'), then entry point will be just ./index